Assets/VRTK/Scripts/Interactions/Highlighters/VRTK_OutlineObjectCopyHighlighter.cs

```diff
@@ -21,6 +21,8 @@ public class VRTK_OutlineObjectCopyHighlighter : VRTK_BaseHighlighter
         public GameObject customOutlineModel;
         [Tooltip("A path to a GameObject to find at runtime, if the GameObject doesn't exist at edit time.")]
         public string customOutlineModelPath = "";
+        [Tooltip("If your mesh has multiple submeshes you wish to highlight, you'll want to check this otherwise only the first mesh will be highlighted")]
```

Comment:
Don't use words like `you` and `you'll` refer like:

`If the mesh has multiple sub-meshes to highlight then this should be checked, otherwise only the first mesh will be highlighted.`